ACE/ace/Auto_Ptr.h

```diff
@@ -70,7 +70,7 @@ ACE_END_VERSIONED_NAMESPACE_DECL
 
 #if !defined (ACE_LACKS_AUTO_PTR)
 #include <memory>
-using std::auto_ptr;
+using std::unique_ptr;
```

Comment:
Let us not pollute the global namespace any further, so only do auto_ptr

Reply:
Then this header should be eliminated at all?

Reply:
Yes, at some point, but that would break any project still using this header so we shouldn't do that at this moment

Reply:
So I shall remove any include of this header in ACE_TAO?

Reply:
Yes, at the moment it is not used anymore it can be removed, make sure memory is included, could be that this is not done in all places. We can't remove the header from the git repo completely, that will break user code

Reply:
on github CI not all examples/tests are compiled, so compile everything locally before you push

Reply:
on OSX not all examples tests compiles too!

ACE_Auto_Basic_Array_Ptr or ACE_Auto_Array_Ptr stil used:

./ace/Refcounted_Auto_Ptr.h
./ace/Auto_Ptr.cpp
./ace/CDR_Stream.cpp
./ace/Configuration.cpp
./ace/DLL_Manager.cpp
./ace/Local_Memory_Pool.cpp
./ace/Log_Record.cpp
./ace/Process.cpp
./ace/Remote_Name_Space.cpp
./ace/SOCK_SEQPACK_Association.cpp
./ACEXML/common/HttpCharStream.cpp
./ACEXML/common/URL_Addr.cpp
./ACEXML/common/XML_Codecs.cpp
./apps/gperf/src/Key_List.cpp
./examples/IPC_SAP/UPIPE_SAP/ex2.cpp
./examples/IPC_SAP/UPIPE_SAP/ex3.cpp
./examples/Web_Crawler/HTTP_URL.cpp
./netsvcs/lib/Client_Logging_Handler.cpp
./netsvcs/lib/Name_Handler.cpp
./protocols/ace/INet/HTTP_BasicAuthentication.cpp
./protocols/ace/HTBP/HTBP_Inside_Squid_Filter.cpp
./protocols/ace/HTBP/HTBP_Channel.cpp
./protocols/ace/HTBP/HTBP_Session.cpp
./protocols/ace/INet/BidirStreamBuffer.h
./protocols/ace/INet/BufferedStreamBuffer.h
./tests/CDR_File_Test.cpp
./tests/CDR_Test.cpp
./tests/Codecs_Test.cpp
./tests/Log_Msg_Test.cpp
./tests/Message_Queue_Test_Ex.cpp
./tests/Reactor_Remove_Resume_Test.cpp
./tests/Reactor_Fairness_Test.cpp
./tests/SString_Test.cpp

Reply:
I can't do more to help, sorry!